serde/se.py

```diff
@@ -381,8 +381,9 @@ def serializable_to_obj(object: Any) -> Any:
         elif is_bearable(o, dict):
             return {k: thisfunc(v) for k, v in o.items()}
         elif is_str_serializable_instance(o) or is_datetime_instance(o):
+            se_cls = o.__class__ if not c or c is Any else c
             return CACHE.serialize(
-                c or o.__class__,
+                se_cls,
```

Comment:
If `Any` is passed in this argument, it will be an infinite recursion?

Reply:
Yes